x-pack/plugins/security_solution/public/cloud_security_posture/components/csp_details/misconfiguration_findings_details_table.tsx

```diff
@@ -102,12 +108,12 @@ export const MisconfigurationFindingsDetailsTable = memo(
 
     const navToFindings = useNavigateFindings();
 
-    const navToFindingsByRuleAndResourceId = (ruleId: string, resourceId: string) => {
-      navToFindings({ 'rule.id': ruleId, 'resource.id': resourceId });
+    const navToFindingsByHostName = (hostName: string) => {
```

Comment:
I suspect this will break the logic for navigating using a `user.name` entity instead of `host.name`
@animehart am I right to assume that? 
We should have tests to make sure both of those navigations work properly. I would expect a test to fail after this change.

Reply:
I didn't intend to push these changes. I revert them now.